content/en/logging/event_stream.md

```diff
@@ -0,0 +1,158 @@
+---
+type: docs
+category: Transparency Log
+menuTitle: Event Stream
+title: Using the Rekor Event Stream
+weight: 1850
+---
+
+The public Rekor instance provides an event stream of new entries added to the transparency log using GCP Pub/Sub. This can be used to monitor the log in real-time for events you are interested in.
+
+## Pub/Sub details
+
+> **Important:** Pub/Sub usage is not free. Please familiarize yourself with the [pricing](https://cloud.google.com/pubsub/pricing) before proceeding.
+
+> **Tip:** You can avoid paying egress network costs by placing the workload that processes events from the subscription in the same region as the topic.
+
+The details needed to subscribe to the topic are below. Any authenticated GCP principal can create subscriptions to the topic.
+
+| Item                 | Value           |
+|----------------------|-----------------|
+| GCP Project ID       | `project-rekor` |
+| Pub/Sub Topic ID     | `new-entry`     |
+| Pub/Sub Topic Region | `us-central1`   |
+
+## Event format
+
+The events are published in both the [JSON](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/json-format.md) and [Protobuf](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/protobuf-format.md) formats of the CloudEvents specificaion. The `data` field is always a serialized [`TransparencyLogEntry`](https://github.com/sigstore/protobuf-specs/blob/b46b842040854ceab8f3a42547ae6e991793d0ef/protos/sigstore_rekor.proto#L92) proto message. For JSON formatted events, this follows the JSON serialization [specification](https://protobuf.dev/programming-guides/proto3/#json) for Protobufs.
```

Comment:
We are only publishing in json currently. I can switch to both if you think we will see adoption of both formats, but I thought json would be more widely used.